dicoveryserver/save_uploaded_info.php

```diff
@@ -10,22 +10,36 @@
 $ipAddress = $_POST['ipAddress'] ?? null;
 
 // Function to save resource
-function save_resource($conn, $userid, $fileName, $fileType, $resourceName, $ipAddress)
+// function save_resource($conn, $userid, $fileName, $fileType, $resourceName, $ipAddress)
+// {
+//     $query = "INSERT INTO `resource` (`resource_name`, `user_id`, `file_type`, `date`, `file_name`, `ip_address`)
+//               VALUES (?, ?, ?, CURDATE(), ?, ?)";
+
+//     $stmt = $conn->prepare($query);
+
+//     $stmt->bind_param("sisss", $resourceName, $userid, $fileType, $fileName, $ipAddress);
+
+//     $result = $stmt->execute();
+
+//     $stmt->close();
+//     return $result;
+// }
+function save_resource($conn, $userid, $ipAddress)
 {
     $query = "INSERT INTO `resource` (`resource_name`, `user_id`, `file_type`, `date`, `file_name`, `ip_address`)
               VALUES (?, ?, ?, CURDATE(), ?, ?)";
 
     $stmt = $conn->prepare($query);
 
-    $stmt->bind_param("sisss", $resourceName, $userid, $fileType, $fileName, $ipAddress);
+    $stmt->bind_param("is",  $userid, $ipAddress);
```

Comment:
what is `sisss` and `is`?

Reply:
The strings "sisss" and "is" in the $stmt->bind_param() function in PHP represent the data types of the parameters being bound to the prepared SQL statement,

"sisss" indicates the data types for five parameters in the order they appear:
s: The first parameter is a string ($resourceName).
i: The second parameter is an integer ($userid).
s: The third parameter is a string ($fileType).
s: The fourth parameter is a string ($fileName).
s: The fifth parameter is a string ($ipAddress).